Python/Tests/TestData/DjangoDebugProject/DjangoDebugProject.pyproj

```diff
@@ -11,7 +11,7 @@
     </SearchPath>
     <WorkingDirectory>.</WorkingDirectory>
     <LaunchProvider>Django launcher</LaunchProvider>
-    <WebBrowserUrl></WebBrowserUrl>
+    <WebBrowserUrl>http://localhost</WebBrowserUrl>
```

Comment:
Probably. Do we still have hope that test machines will be able to handle the UI tests? ;)
At least one test using this project is setting a breakpoint and launch debugging, and waits for the breakpoint to hit.

---
In reply to: [107991880](https://github.com/Microsoft/PTVS/pull/2323#discussion_r107991880) [](ancestors = 107991880)